src/components/NFTViewOptionList.vue

```diff
@@ -38,70 +38,49 @@
     </p>
 
     <template v-if="contentUrls.length && shouldShowContentUrlButtons">
-      <template v-if="hasDuplicatedContentTypes">
```

Comment:
Pls remove the computed property too

Reply:
[🔥 Remove unused computed](https://github.com/likecoin/liker-land/pull/1635/commits/a05c9c1ebc21fe676f2d480e3472562d3228a87d)